ci-operator/step-registry/ipi/conf/ibmcloud/provisionednetwork/ipi-conf-ibmcloud-provisionednetwork-commands.sh

```diff
@@ -6,9 +6,20 @@ set -o pipefail
 
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
+vpcSubnetsFile="${SHARED_DIR}/customer_vpc_subnets.yaml"
 
-cat "${SHARED_DIR}/customer_vpc_subnets.yaml"
+cat "${vpcSubnetsFile}"
+region="${LEASED_RESOURCE}"
+declare -a zones=("${region}-1" "${region}-2" "${region}-3")
 
-yq-go m -x -i "${CONFIG}" "${SHARED_DIR}/customer_vpc_subnets.yaml"
+if [[ ${ZONES_COUNT} -eq 1 ]]; then
```

Comment:
When `ZONES_COUNT` ==  2, what would happen ?

Reply:
I have not found method to deal with the vpcSubnetsFile

Reply:
I wonder what is the issue for the original script before this change ? If you plan to introduce `ZONES_COUNT` in the step, you have to take other values into consideration, too, but not only `ZONES_COUNT` == 1 case.